### PR TITLE
Make it easier to deploy with argo

### PR DESCRIPTION
**Pull request by gparvin**
Adding a simple script to help make it easy to deploy policies with argocd.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ command can be viewed in its [README](deploy/README.md).) This script assumes yo
 Application lifecycle management as an addon in your Open Cluster Management installation. See
 [Application lifecycle management](https://open-cluster-management.io/getting-started/integration/app-lifecycle/) 
 for details on installing the Application addon.
+**Note**: If you are using ArgoCD for gitops, a similar script `argoDeploy.sh` is provided that does
+not require the Application lifecycle addon.
```
Comment (mprahl) [inline README.md]:
```suggestion
not require the Application Lifecycle addon.
```

File: README.md [hunk 2]
```diff
@@ -46,6 +46,8 @@ command can be viewed in its [README](deploy/README.md).) This script assumes yo
 Application lifecycle management as an addon in your Open Cluster Management installation. See
 [Application lifecycle management](https://open-cluster-management.io/getting-started/integration/app-lifecycle/) 
 for details on installing the Application addon.
+**Note**: If you are using ArgoCD for gitops, a similar script `argoDeploy.sh` is provided that does
```
Comment (mprahl) [inline README.md]: Could you link `argoDeploy.sh`?

File: deploy/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Deploy policies to Open Cluster Management
 
-Deploy policies to Open Cluster Management with the `deploy.sh` script.
+Deploy policies to Open Cluster Management with the `deploy.sh` script. The `deploy.sh` script uses application lifecycle subscriptions for deployment.  To use ArgoCD use the `argoDeploy.sh` script instead.
```
Comment (mprahl) [inline deploy/README.md]:
```suggestion
Deploy policies to Open Cluster Management with the `deploy.sh` script. The `deploy.sh` script uses Application Lifecycle subscriptions for deployment.  To use ArgoCD use the `argoDeploy.sh` script instead.
```

**Review by mprahl — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (gparvin) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (gparvin) [reply README.md]: Done

File: deploy/README.md (hunk 3 above)
Comment (gparvin) [reply deploy/README.md]: Done

**Review by mprahl — APPROVED** [review]
/unhold

**Pull request merged**